src/relay/blockchain/events_informations.py

```diff
@@ -1,20 +1,26 @@
 import math
-from collections import namedtuple
 from typing import List
 
+import attr
+
 from relay.blockchain.currency_network_events import (
     BalanceUpdateEvent,
     BalanceUpdateEventType,
     TrustlineUpdateEventType,
 )
 from relay.network_graph.interests import calculate_interests
 
-InterestAccrued = namedtuple("InterestAccrued", ["value", "interest_rate", "timestamp"])
-
 DIRECTION_SENT = "sent"
 DIRECTION_RECEIVED = "received"
 
 
+@attr.s
+class InterestAccrued(object):
```

Comment:
There's no need to derive from `object` here. Please change it to not derive from `object`.